Kristian-Repizo/tarea-2-form/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
+    <title>Form</title>
+  </head>
+  <body>
+    <main class="container">
+      <form>
+        <section>
+            <label class="title-search">Complete the form</label><br />
+        </section>
+
+        <section class="name">
+          <label for="fname">Name:</label>
+          <input id="fname" type="text" name="name" required="" />
+        </section>
+
+        <section>
+          <label for="sname">Last name:</label>
+          <input id="sname" type="text" name="surname" required="" />
+        </section>
+
+        <section>
+          <label for="xage">Age:</label>
+          <input id="xage" type="number" name="age" min="0" max="50" />
+        </section>
+
+        <section>
+          <label for="xbirth">Date of birthday:</label>
+          <input id="xbirth" type="date" name="datebirth" />
+        </section>
+
+        <section>
+          <label for="xcolor">Favorite color:</label>
+          <input class="cursor" id="xcolor" type="color" name="fvcolor" />
+        </section>
+
+        <section>
+          <label for="xsport">Do you practice sports?</label>
+        </section>
+
+        <article id="yes-not">
```

Comment:
@Kristian-Repizo the article is meant to enclose a more generic content, so this one should enclose all form section even the form itself. If you need to differentiate the yes-not form controls you use another section.
It's fine to have section inside sections, even check the fieldset tag.